### PR TITLE
added TenantEntity, and made some minor changes

### PR DESCRIPTION
**Pull request by grigor-yeghiazaryan**

### CODE REVIEW

**Review by vardkhachatryan96 — APPROVED** [review]

File: src/TourManager/TourManager.Service.Model/Activity.cs [hunk 1]
```diff
@@ -12,7 +12,7 @@ public class Activity
 		public string Phone { get; set; }
 		public string Email { get; set; }
 		public string Website { get; set; }
-		public ActivityOption ActivityType { get; set; }
+		public ActivityType ActivityType { get; set; }
```
Comment (TomaCode) [inline src/TourManager/TourManager.Service.Model/Activity.cs:15]: we can already rename also property to just Type, will be public ActivityType Type { get; set; }

**Review by TomaCode — APPROVED** [review]

File: src/TourManager/TourManager.Service.Model/Activity.cs (hunk 1 above)
Comment (TomaCode) [reply src/TourManager/TourManager.Service.Model/Activity.cs:15]: same applies to other properties, PropertyType.....etc

Comment (grigor-yeghiazaryan) [reply src/TourManager/TourManager.Service.Model/Activity.cs]: done

Comment (HAghajanyan-ST) [reply src/TourManager/TourManager.Service.Model/Activity.cs]: I think on the end we will get all this data from appropriate service and we will delete it from our side, to be more flexible. But for now its ok.

**Review by HAghajanyan-ST — APPROVED** [review]

**Review by TomaCode — APPROVED** [review]

**Pull request merged**